### PR TITLE
Publish final schedule

### PR DESCRIPTION
**Pull request by chrisguttandin**
This pull request adds the final schedule. Design-wise there is definitely room for improvement, but it does at least contain all the information.

The program page has been removed as all the infos are now in the schedule.

### CODE REVIEW

File: src/layouts/index.js [hunk 1]
```diff
@@ -13,7 +13,6 @@ const menuItems = [
   { link: '/', text: 'Home', exact: true },
   { link: 'https://ti.to/web-audio-conference/wac-2018', external: true, text: 'Tickets', extraClassName: 'header__emph' },
   { link: '/schedule', text: 'Schedule' },
-  { link: '/program', text: 'Program' },
```
Comment (janmonschke) [inline src/layouts/index.js]: Could you add a redirect from `/program` to `/schedule` ?

Comment (chrisguttandin) [reply src/layouts/index.js]: Good idea @janmonschke, do you know how this works in Gatsby? I could duplicate the schedule page at the /program route but that doesn't feel right.

Comment (chrisguttandin) [reply src/layouts/index.js]: I found out how to do it. It's pretty simple if you know what to google for. :-)

File: gatsby-node.js [hunk 2]
```diff
@@ -4,4 +4,58 @@
...
+  acceptedSubmissions.talks
+    .forEach((talk) => {
+      boundActionCreators.createPage({
+        component: resolve(`./src/components/accepted-submission.js`),
+        context: { ...talk, type: 'talk' },
+        path: `/presentations/${ talk.slug }`
+      });
+    });
+
+  boundActionCreators.createRedirect({ fromPath: '/program', isPermanent: true, toPath: '/schedule', redirectInBrowser: true });
```
Comment (janmonschke) [inline gatsby-node.js]: 👍

**Comment by janmonschke** [comment]
I think you can merge it since Tom is on vacation and won't have the time to check it out

**Pull request merged**